private/bufpkg/bufimage/bufimagebuild/builder.go

```diff
@@ -81,132 +79,73 @@ func (b *builder) build(
 		paths[i] = targetFileInfo.Path()
 	}
 
-	buildResults := getBuildResults(
+	buildResult := getBuildResult(
 		ctx,
 		parserAccessorHandler,
 		paths,
-		excludeSourceCodeInfo,
+		options,
 	)
-	var buildResultErr error
-	for _, buildResult := range buildResults {
```

Comment:
Much of the red lines in this file are because `protocompile` natively handles concurrency. So a lot of logic in here that merged chunks of built results is no longer needed.